bcgov/invasivesbc
# Add legends button and popup

# Overview

This PR includes the following proposed change(s):

- Provides the implementation for legends Popup #569
- Added a button on the map. At a click of a button, a popup showing information regarding legends is displayed.
- The user can close the popup using the close button on the top left corner or press the escape button.

## Type of change

- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add a `breaking change` label to this PR

## How Has This Been Tested?

- Ensure the popup opens and closes as expected using the close button and escape button
- Ensure the content is properly visible on smaller screens 
- Ensure the formatting of tables is correct
- Ensure the Map Legends button works as expected and the correct tooltip is shown

## Screenshots

![image](https://user-images.githubusercontent.com/8287486/229865661-60670967-c0df-4964-aa4c-62d75ffa8cff.png)

![image](https://user-images.githubusercontent.com/8287486/229865964-fe246835-3c02-41ae-bb52-365229b3b8a9.png)

![image](https://user-images.githubusercontent.com/8287486/229866062-80bf7cbd-e3ac-43c8-8efa-d133da4efb86.png)

![image](https://user-images.githubusercontent.com/8287486/229866189-58d31e57-fd26-4cea-ad5b-4663a573bfc7.png)


## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have made corresponding changes to the documentation
- [ x] New and existing unit tests pass locally with my changes
